.changeset/gorgeous-nails-film.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
+---
+
+Fixes a bug where `pnpm` was not detected correctly when detecting compile packages
```

Comment:
```suggestion
Fixes a bug where `pnpm` was not detected correctly when detecting compiled packages
```

perhaps?

Reply:
I went for **compile** since we are the ones that compile the packages, rather then detecting already **compiled** packages.

Reply:
In that case, might be worth adding a link.
```suggestion
Fixes a bug where `pnpm` was not detected correctly when detecting [compile packages](https://seek-oss.github.io/sku/#/./docs/extra-features?id=compile-packages)
```